### DISCUSSION TITLE
Allow/present custom fields within directus_fields + directus_collections

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#3020] [Feature Requests]
We're planning to support extending all system fields in Directus 9, and while we focus mostly on Users and Files, there are good use-cases for those extending Directus Collections and Directus Fields (etc) with additional metadata. While this should work as expected in the API, it would be nice to also include this in the App.. perhaps within an additional/conditional pane of the settings modal.

**Comment by benhaynes** [comment]
@shinroo and @cheribhai 🔔

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
